.github/workflows/ci.yml

```diff
@@ -22,26 +22,35 @@ jobs:
           --health-interval 10s
           --health-timeout 5s
           --health-retries 5
+    strategy:
+      matrix:
+        otp: ['24.3.4']
+        elixir: ['1.14.1']
     steps:
     - uses: actions/checkout@v2
     - name: Set up Elixir
-      uses: erlef/setup-elixir@885971a72ed1f9240973bd92ab57af8c1aa68f24
+      uses: erlef/setup-beam@v1
       with:
-        elixir-version: '1.12.3' # Define the elixir version [required]
-        otp-version: '24.0.2' # Define the OTP version [required]
-    - name: Restore dependencies cache
-      uses: actions/cache@v2
+        otp-version: ${{ matrix.otp }}
```

Comment:
BTW: don't really see a point to using this `${{ matrix.otp }}` if we're only testing with _one_ version of `OTP` ... 💭